util/raclgen.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""RACL Generator
```

Comment:
Add proper description about what the tool does, i.e., that it computes the policy selection vector for a given config and ip